web/src/main/java/org/cbioportal/web/util/appliers/TreatmentRowExtractor.java

```diff
@@ -1,19 +1,17 @@
 package org.cbioportal.web.util.appliers;
 
+import org.cbioportal.model.ClinicalEventSample;
 import org.cbioportal.model.TreatmentRow;
 import org.springframework.stereotype.Service;
 
-import java.util.HashSet;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 @Service
 public class TreatmentRowExtractor {
     public Set<String> extractSamples(TreatmentRow row) {
-        Set<String> samples = new HashSet<>();
-        row.getSamples().forEach(sample -> {
-            samples.add(sample.key());
-        });
-
-        return samples;
+        return row.getSamples().stream()
```

Comment:
This logic hasn't changed in practice. The collection is immutable now, but that doesn't matter. No idea why I made this code so gross to begin with.